CommandLine.Tests/HelpViewTests.cs

```diff
@@ -303,5 +303,36 @@ public void When_unmatched_tokens_are_allowed_then_help_view_indicates_it()
 
             helpView.Should().Contain("Usage: some-command [options] [[--] <additional arguments>...]]");
         }
+
+        [Fact]
+        public void Retain_single_dash_on_multi_char_option_when_display_raw_aliases_is_specified()
+        {
+            var command = Command("command", "Help Test",
+                    Option("-multi|--alt-option", "Help for option"));
+            HelpViewOptions helpViewOptions = new HelpViewOptions(true);
+            var helpView = command.HelpView(helpViewOptions);
+            helpView.Should().Contain("-multi");
+            helpView.Should().NotContain("--multi");
+        }
```

Comment:
It will display --multi without my change (just locally modified the unit test). I can withdraw this PR and make a corresponding bug fix, if that's your preference. It'd just be a matter of using my version of HelpViewExtensions.LeftColumnText() which uses the raw aliases.. The existing one infers the number of dashes from the length of the option text.